cpp/arcticdb/util/error_code.hpp

```diff
@@ -46,6 +47,7 @@ inline std::unordered_map<ErrorCategory, const char*> get_error_category_names()
         {ErrorCategory::SORTING, "SORTING"},
         {ErrorCategory::USER_INPUT, "USER_INPUT"},
         {ErrorCategory::COMPATIBILITY, "COMPATIBILITY"},
+        {ErrorCategory::STORAGE_RETRYABLE, "STORAGE_RETRYABLE"},
```

Comment:
The new error category. `StorageRetryableException` is registered as it is in the above. However, in  
https://github.com/man-group/ArcticDB/blob/1b925357c1cabd734d1654f838afc3e8111b1275/python/arcticdb/util/errors.py#L41, the name generated, based on this line, is `Storage_retryableException`. 
Suggest changing them to `STORAGERETRYABLE` and `StorageretryableException` instead.